packages/core/addon/helpers/ember-array.js

```diff
@@ -0,0 +1,9 @@
+import { helper } from '@ember/component/helper';
+import { A as EmberArray } from '@ember/array';
+
+export function emberArray(params) {
+  // slice params to avoid mutating the provided params
+  return EmberArray(params.slice());
```

Comment:
What if `params` is `null|undefined`

Reply:
Since this was meant to be an alias for the `array` helper from ember-composable-helpers, I've just exported that helper directly.